draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -818,26 +818,26 @@ chosen in MP_SEQ are considered sufficiently large.
 {: #ref-MP_HMAC title='Format of the MP_HMAC Suboption'}
 
 The MP_HMAC suboption is used to provide authentication for the MP_JOIN,
-MP_ADDADDR, and MP_REMOVEADDR suboptions. The HMAC code is generated according
+MP_ADDADDR, and MP_REMOVEADDR suboptions. For this specification of MP-DCCP, the HMAC code is generated according
 to {{RFC2104}} in combination with the SHA256 hash algorithm described in
 {{RFC6234}}, with the output truncated to the leftmost 160 bits (20 bytes).
 
 The "Key" used for the HMAC computation is the derived key (d-key)
-described in {{MP_KEY}}, while the HMAC "Message" is a concatenation of
+described in {{MP_KEY}}, while the HMAC "Message" for MP_JOIN, MP_ADDADDR and MP_REMOVEADDR is a concatenation of:
 
-   * MP_JOIN: The CI and nonce of the MP_JOIN for which authentication
+   * for MP_JOIN: The CI and nonce of the MP_JOIN for which authentication
```

Comment:
When we discussed this part in the telco last week, we decided to also add the psudocode style description. Not sure if it was intentionally left out or forgotten. Also not sure if it is really needed, so just putting a reminder of the discussion :)

Reply:
That is correct. For example the one for the MP_JOIN can be found in section 4.3:
`MP_HMAC(B) = HMAC-SHA256(Key=d-key(B), Msg=RB+RA)`

Reply:
This example "MP_HMAC(B) = HMAC-SHA256(Key=d-key(B), Msg=RB+RA)" and the description of the HMAC calculation for an MP_JOIN in section 4.3, don't match with the description written here. Here it is stated that  Msg = CI(A or B)+R(A or B),

Reply:
I think CI is not needed and I removed it from the calculation. I also added HMAC-SHA265 calculation examples for MP_ADDADDR and MP_REMOVEADDR in https://github.com/tsvwg/ietf-multipath-dccp/pull/280/commits/23f89ad63eaf38c65e3d5afff15a8c55ddde75eb
@abrunstrom @NathalieRM